rfcs/20181029-more-flexible-concurrency.md

```diff
@@ -0,0 +1,95 @@
+# More Flexible Concurrency
+
+| Status        | (Proposed)                                           |
+:-------------- |:---------------------------------------------------- |
+| **Author(s)** | Gabriel Perdue (gnperdue@gmail.com),                 |
+|               | Christopher Jones (cdj@fnal.gov),                    |
+|               | Matti Kortelainen (matti@fnal.gov)                   |
+| **Sponsor**   | Ravi Chirravuri (crk@google.com)                     |
+| **Updated**   | 2018-10-29                                           |
+
+## Objective
+
+We would like to implement a more flexible thread scheduling system. This would
```

Comment:
nit: This document presents a proposal to implement a more flexible thread scheduling system.